src/Hydration/IssueProjectHydration.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace DanielBadura\Redmine\Api\Hydration;
+
+use DanielBadura\Redmine\Api\Client;
+use DanielBadura\Redmine\Api\Struct\Issue;
+use ProxyManager\Factory\LazyLoadingValueHolderFactory;
+use ProxyManager\Proxy\LazyLoadingInterface;
+
+/**
+ * @author Marco Giesen <marco.giesen93@gmail.com>
+ */
+class IssueProjectHydration
+{
+    /**
+     * @param Issue $issue
+     * @param Client $client
+     * @return Issue
+     */
+    public function hydrateOneIssue(Issue $issue, Client $client)
+    {
+        $project = $issue->getProject();
+        $projectRepository = $client->getProjectRepository();
+
+        $factory = new LazyLoadingValueHolderFactory();
+        $initializer = function (
+            &$wrappedObject,
+            LazyLoadingInterface $proxy,
+            $method
+        ) use ($project, $projectRepository) {
+            if($method == "getIdentifier" ||
+                $method == "getDescription" ||
+                $method == "getCreatedOn" ||
+                $method == "getUpdatedOn" ||
+                $method == "isIsPublic" ||
```

Comment:
auto generated code, the value itself is "isPublic", named by redmine. I will rename it to simply "isPublic"